spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -502,6 +503,14 @@ object TimestampNTZTableFeature extends ReaderWriterFeature(name = "timestampNtz
   }
 }
 
+object VariantTypeTableFeature extends ReaderWriterFeature(name = "variantType")
```

Comment:
```suggestion
object VariantTypeTableFeature extends ReaderWriterFeature(name = "variantType-dev")
```
(the protocol RFC has not been accepted yet)

Reply:
added the "-dev" suffix back